integration-tests/src/test/resources/features/end2endtestit.feature

```diff
@@ -3,9 +3,8 @@ Feature: End to End Bridge integration tests
   Scenario:By default Manager url should not be accessible without authentication
     Given get list of Bridge instances fails with HTTP response code 401
 
-
   Scenario: Bridge is created and in ready state
-    Given get list of Bridge instances with access token doesn't contain randomly generated Bridge
```

Comment:
Why did you delete this step?

Reply:
the number generation is done in `create a Bridge with randomly generated name with access token`

Reply:
ok